src/components/common/Input/index.js

```diff
@@ -3,17 +3,20 @@ import PropTypes from 'prop-types';
 import { View, TextInput, Text } from 'react-native';
 import styles from './styles';
 
-const Input = ({ input: { onChange, ...restInput }, password = false, label, meta: { touched, error } }) => (
+const Input = ({ input: { onChange, ...restInput }, placeholder = '', autoCapitalize = 'none', password = false, keyboardType = 'default', label, meta: { touched, error } }) => (
```

Comment:
Maybe you could divide this in more lines so the line doesn't end up being that long.